superset-frontend/src/components/Menu/SubMenu.tsx

```diff
@@ -83,8 +97,8 @@ export interface ButtonProps {
 
 export interface SubMenuProps {
   buttons?: Array<ButtonProps>;
-  name: string;
-  children?: MenuChild[];
+  name?: string;
+  tabs?: MenuChild[];
```

Comment:
Why change it from `children` to `tabs`? Passing props kind of goes against React's declarative nature.